lobster/tools/codebeamer/codebeamer.py

```diff
@@ -50,6 +50,8 @@
 from lobster.errors import Message_Handler, LOBSTER_Error
```

Comment:
Currently the token can ONLY be provided in the configuration file. But in `README.md` you write that it can be provided as a command line argument, too.